packages/formation/sass/modules/_m-megamenu.scss

```diff
@@ -18,7 +18,6 @@ $marketing-container-height: 380px;
     a {
       padding: 20px 25px 20px 7px;
       font-weight: bold;
-      width: 111%;
```

Comment:
@accbjt Do you know of any side effects that removing this will have?

Reply:
Does this look ok on mobile? That'd be my only guess.

Reply:
It doesn't appear to have any impact on mobile, because the width of the list item already extends past the width of the container lol.

![image](https://user-images.githubusercontent.com/1915775/53597332-e3c11000-3b6f-11e9-920c-9be1f2f07fd9.png)